profileservice/api/controllers/profiles.py

```diff
@@ -13,16 +13,15 @@
 #  limitations under the License.
 
 import json
-import datetime
+import hmac
 import logging
 import uuid as uuidlib
 import copy
 
-from flask import jsonify, request, g
+from flask import request, g
 from bson import ObjectId
 
 import controllers.configs as cfg
-import utils.mongoutils as mongoutils
```

Comment:
Is this line removal intentional? Seems like it could cause issues with lines calling 'mongoutils' below.

Reply:
there are two lines of this, so I removed this one.